static/src/javascripts/projects/common/modules/business/stocks.js

```diff
@@ -0,0 +1,60 @@
+define([
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/ajax',
+    'common/utils/config',
+    'common/utils/template',
+    'text!common/views/business/stock-value.html',
+    'text!common/views/business/stocks.html'
+], function (
+    $,
+    _,
+    ajax,
+    config,
+    template,
+    stockValueTemplate,
+    stocksTemplate
+) {
+    function isBusinessFront() {
+        return config.page.pageId === config.page.edition.toLowerCase() + '/business';
+    }
+
+    function getStocksData(onSuccess) {
+        ajax({
+            url: '/business-data/stocks.json',
+            type: 'json',
+            method: 'get',
+            crossOrigin: true,
+            success: onSuccess
+        });
```

Comment:
You should return the ajax promise form this function rather than passing around callbacks.